ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -102,6 +104,8 @@ const getDescription = (
         return t('confirmTitleDescApproveTransaction');
       }
       return t('confirmTitleDescERC20ApproveTransaction');
+    case TransactionType.tokenMethodIncreaseAllowance:
```

Comment:
[nit] `TransactionType.tokenMethodApprove` and `TransactionType.tokenMethodIncreaseAllowance` cases can be moved to make the switch statement alphabetical. Same comment for the switch statement in getTitle above